tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -258,6 +258,29 @@ public void EnsureContainerWithCustomEntrypointEmitsEntrypoint()
         Assert.Equal("custom", entrypoint.GetString());
     }
 
+    [Fact]
+    public void EnsureAllNatsManifestTypesHaveVersion0Suffix()
```

Comment:
Remove this test and turn it into a test like this https://github.com/dotnet/aspire/blob/1f8e53ddd36a0f03c6fdfb9b5bfc8e7810178905/tests/Aspire.Hosting.Tests/Kafka/AddKafkaTests.cs#L69-L78